openshift/templates/wfwx_noon_forecasts.cronjob.yaml

```diff
@@ -20,7 +20,7 @@ parameters:
   - name: IMAGE_REGISTRY
     value: image-registry.openshift-image-registry.svc:5000
   - name: JOB_NAME
-    value: bcfw-p1-forecasts
+    value: wfwx-noon-forecasts
```

Comment:
Remember to take care when pushing to production, to manually delete the bcfw-p1-forecasts-wps-prop